package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/node-jspm-jasmine.js",
   "scripts": {
     "test": "test",
-    "build": "babel src --out-dir lib",
+    "build": "./node_modules/babel-cli/bin/babel.js src --out-dir lib",
```

Comment:
In an effort to force npm to use local versions, I must have thought otherwise at some point, and the notion persisted, hehe. Sorry about that, thanks for pointing it out. I'll revert.